client/landing/stepper/declarative-flow/internals/steps-repository/components/accent-color-control/index.tsx

```diff
@@ -32,40 +33,60 @@ enum COLORS {
 	VividPurple = '#9B51E0',
 }
 
-const COLOR_OPTIONS = [
-	{
-		label: 'Lettre',
-		value: COLORS.Lettre,
-		icon: <ColorSwatch color={ COLORS.Lettre } />,
-	},
-	{
-		label: 'Black',
-		value: COLORS.Black,
-		icon: <ColorSwatch color={ COLORS.Black } />,
-	},
-	{
-		label: 'Vivid red',
-		value: COLORS.VividRed,
-		icon: <ColorSwatch color={ COLORS.VividRed } />,
-	},
-	{
-		label: 'Vivid purple',
-		value: COLORS.VividPurple,
-		icon: <ColorSwatch color={ COLORS.VividPurple } />,
-	},
-	{
-		label: 'Custom',
-		value: 'custom',
-		icon: <Icon className="custom_color_icon" icon={ color } width={ 22 } height={ 22 } />,
-	},
-];
+const getColorOptions = ( shouldLimitGlobalStyles: boolean, groupLabelText: string ) => {
+	const locked = shouldLimitGlobalStyles && {
+		extra: <Gridicon icon="lock" size={ 18 } className="extra-gridicon" />,
```

Comment:
Let's use the `lock` icon from `@wordpress/icons`

E.g.
`<Icon icon={ lock } width={ 22 } height={ 22 } />`
(width and height copied from the other `Icon` in this component, so they might not be perfect for the lock)

Reply:
Why should we use the `Icon` component instead of the `GridIcon` component? I've looked at the Calypso docs and they [only reference the `Gridicon` component when working with icons](https://wpcalypso.wordpress.com/devdocs/docs/icons.md).

Reply:
Mmh, I don't think there's a specific doc about it, but we've been internally talking about preferring WordPress Icons over other ones whenever possible.
An example: PuPv3-aHW-p2

In this case, we have an exact match, and the `@wordpress/icons` is already imported into the file, so it should be pretty _straightforward_! 🙂

Reply:
Hmmm, I'm having some trouble with this. If I use `Icon` lock I'm getting some console errors.

`Uncaught TypeError: window.AppBoot is not a function`

If I try other icons it works fine 🤔